lb.tf

```
@@ -126,8 +126,8 @@ resource "exoscale_compute" "lb" {
     ],
     "runcmd" = [
       "while lsof -F p /var/lib/dpkg/lock 2>/dev/null; do echo \"Waiting for dpkg lock...\"; sleep 15; done",
-      "curl -Lo /tmp/ursula.deb ${trimsuffix(var.bootstrap_bucket, "/")}/ursula.deb",
-      "dpkg -i /tmp/ursula.deb",
+      "curl -Lo /tmp/floaty.deb https://github.com/vshn/floaty/releases/download/v1.0.1/floaty_linux_amd64.deb",
```

Automatically fetching the latest release would be possible with

```suggestion
      "curl -Lo /tmp/floaty.deb https://github.com/vshn/floaty/releases/latest/download/floaty_linux_amd64.deb",
```

---

TIL something new :)